integration-tests/src/test.rs

```diff
@@ -0,0 +1,472 @@
+#[cfg(test)]
+mod test {
+
+    use axelar_wasm_std::nonempty;
+    use connection_router::state::ChainName;
+    use cosmwasm_std::{coins, Addr, Binary, Deps, Env, HexBinary, StdResult, Uint128, Uint256};
+    use cw_multi_test::{App, ContractWrapper, Executor};
+
+    use multisig::key::PublicKey;
+
+    const AXL_DENOMINATION: &str = "uaxl";
+    #[test]
+    fn test() {
```

Comment:
This is a bad name.

Reply:
lol this is just a placeholder to show how the setup of the testing environment works. It's not really a test. But I can change it to something a little better for now anyways.